ostree-vsphere.sh

```diff
@@ -0,0 +1,790 @@
+
+##################################################################
+##
+## Upload image to datastore
+##
+##################################################################
+greenprint "📋 Uploading vmdk image to vsphere datacenter 7.0"
+govc import.vmdk -dc="${DATACENTER_70}" -ds="${DATASTORE_70}" -pool="${DATACENTER_70_POOL}" "${VMDK_FILENAME}" > /dev/null
+greenprint "📋 Copying vmdk image to vsphere datacenter 6.7"
+govc datastore.mkdir -dc="${DATACENTER_67}" -ds="${DATASTORE_67}" "${COMPOSE_ID}-image"
+govc datastore.cp -dc="${DATACENTER_70}" -ds="${DATASTORE_70}" -dc-target="${DATACENTER_67}" -ds-target="${DATASTORE_67}" "${COMPOSE_ID}-image/${VMDK_FILENAME}" "${COMPOSE_ID}-image/${VMDK_FILENAME}" > /dev/null
+
+##################################################################
+##
+## Create vm on datacenter7.0-amd and test it
+##
+##################################################################
+# Create vm with vmdk image
+greenprint "📋 Create vm in vsphere datacenter 7.0-AMD"
+DC70_VSPHERE_VM_NAME="${COMPOSE_ID}-70"
+govc vm.create -dc="${DATACENTER_70}" -ds="${DATASTORE_70}" -pool="${DATACENTER_70_POOL}" \
+    -net="VM Network" -net.adapter=vmxnet3 -disk.controller=pvscsi -on=false -c=2 -m=4096 \
+    -g="${GUEST_ID_DC70}" -firmware=efi "${DC70_VSPHERE_VM_NAME}"
+govc vm.disk.attach -dc="${DATACENTER_70}" -ds="${DATASTORE_70}" -vm "${DC70_VSPHERE_VM_NAME}" \
+    -link=false -disk="${COMPOSE_ID}-image/${VMDK_FILENAME}"
+govc vm.power -on -dc="${DATACENTER_70}" "${DC70_VSPHERE_VM_NAME}"
+DC70_GUEST_ADDRESS=$(govc vm.ip -v4 -dc="${DATACENTER_70}" -wait=10m "${DC70_VSPHERE_VM_NAME}")
+greenprint "🛃 Edge VM IP address is: ${DC70_GUEST_ADDRESS}"
+
+# Run ansible check on edge vm
+# Check for ssh ready to go.
+greenprint "🛃 Checking for SSH is ready to go"
+for _ in $(seq 0 30); do
+    RESULTS="$(wait_for_ssh_up "${DC70_GUEST_ADDRESS}")"
+    if [[ $RESULTS == 1 ]]; then
+        echo "SSH is ready now! 🥳"
+        break
+    fi
+    sleep 10
+done
+
+# Check image installation result
+check_result
+
+greenprint "🕹 Get ostree install commit value"
+INSTALL_HASH=$(curl "${PROD_REPO_URL}/refs/heads/${OSTREE_REF}")
+
+tee "${TEMPDIR}"/inventory > /dev/null << EOF
+[ostree_guest]
+${DC70_GUEST_ADDRESS}
+
+[ostree_guest:vars]
+ansible_python_interpreter=/usr/bin/python3
+ansible_user=admin
+ansible_private_key_file=${SSH_KEY}
+ansible_ssh_common_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+ansible_become=yes
+ansible_become_method=sudo
+ansible_become_pass=${IGNITION_USER_PASSWORD}
+EOF
+
+# Test IoT/Edge OS
+podman run --annotation run.oci.keep_original_groups=1 -v "$(pwd)":/work:z -v "${TEMPDIR}":/tmp:z \
+    --rm quay.io/rhel-edge/ansible-runner:latest ansible-playbook -v -i /tmp/inventory \
+    -e ignition="true" \
+    -e os_name=redhat \
+    -e ostree_commit="${INSTALL_HASH}" \
+    -e ostree_ref="${REF_PREFIX}:${OSTREE_REF}" \
+    -e skip_rollback_test="false" \
```

Comment:
We do not have this arg in check-ostree.yaml

Reply:
removed.